modulecmd.tcl.in

```diff
@@ -4036,6 +4063,33 @@ proc renderSettings {} {
          }
       }
    }
+   foreach funcname [array names g_stateFunctions] {
+      switch -- $g_stateFunctions($funcname) {
+         {new} {
+            set val $::g_Functions($funcname)
+            switch -- $::g_shellType {
+               {sh} {
+                  #set val [charEscaped $val]
+		  puts stdout "$funcname () { $val ;} ; export $funcname;"
```

Comment:
I wonder if a function is exported the same way over all sh-kind shells (sh, bash, ksh & zsh). Especially `bash` has a `-f` optional argument to `export` command for function. Isn't it useful here ?